spec/models/comment_spec.rb

```diff
@@ -3,6 +3,17 @@
 describe Comment do
   it_behaves_like "email_confirmable"
 
+  describe "#confirm!" do
+    let(:comment) do
+      VCR.use_cassette('planningalerts') { build(:comment, confirmed: true) }
+    end
+
+    it "should run callback once already confirmed" do
```

Comment:
I'm gonna leave this one to you @henare , I had a quick look at it but couldn't discern what the intention is.